docs/en_US/Compressor/CompressionUtils.md

```diff
@@ -0,0 +1,100 @@
+# Analysis Utils for Model Compression
+We provide several easy-to-use tools for users to analyze their model during model compression.
+
+## Sensitivity
+First, we provide a sensitivity analysis tool (**SensitivityAnalysis**) for users to analyze the sensitivity of each convolutional layer in their model. Specifically, the SensitiviyAnalysis gradually prune each layer of the model, and test the accuracy of the model at the same time. Note that, SensitivityAnalysis only prunes a layer once a time, and the other layers are set to their original weights. According to the accuracies of different convolutional layers under different sparsities, we can easily find out which layers the model accuracy is more sensitive to. 
+
+### Usage
+
+Following codes show the basic usage of the SensitivityAnalysis.
+```python
+from nni.compression.analysis_utils.sensitivity.torch.sensitivity_analysis import SensitivityAnalysis
+
+def val(model):
+    model.eval()
+    total = 0
+    correct = 0
+    with torch.no_grad():
+        for batchid, (data, label) in enumerate(val_loader):
+            data, label = data.cuda(), label.cuda()
+            out = model(data)
+            _, predicted = out.max(1)
+            total += data.size(0)
+            correct += predicted.eq(label).sum().item()
+    return correct / total
+
+s_analyzer = SensitivityAnalysis(model=net, val_func=val)
+sensitivity = s_analyzer.analysis(val_args=[net])
+os.makedir(outdir)
+s_analyzer.export(os.path.join(outdir, filename))
+s_analyzer.visualization(outdir, merge=True)
+s_analyzer.visualization(outdir, merge=False)
+```
+
+Two key parameters of SensitivityAnalysis are model, and val_func. 'model' is the neural network that to be analyzed and the 'val_func' is the validation function that returns the model accuracy on the validation dataset. Due to different scenarios may have different ways to calculate the loss/accuracy, so users should prepare a function that returns the model accuracy on the dataset and pass it to SensitivityAnalysis.
```

Comment:
Depending on the task, the validation metrics maybe other metrics, no necessary to be `accuracy`

Reply:
Currently, we only support the accuracy, because the stop condition for early_stop is to judge if the current metric(for example, accuracy) is lower than original metric - threshold. If we use loss as the metric, we should judge if the current metric is larger than the original metric + threshold, because the losses of the model are growing during the pruning. However, it only needs a negligible effort to support the loss at the same time. I can update the code if needed.